selfdrive/ui/qt/onroad.cc

```diff
@@ -273,9 +273,11 @@ void AnnotatedCameraWidget::updateState(const UIState &s) {
   setProperty("is_cruise_set", cruise_set);
   setProperty("is_metric", s.scene.is_metric);
   setProperty("speed", cur_speed);
+  setProperty("isStandstill", sm["carState"].getCarState().getStandstill());
   setProperty("setSpeed", set_speed);
   setProperty("speedUnit", s.scene.is_metric ? tr("km/h") : tr("mph"));
   setProperty("hideDM", cs.getAlertSize() != cereal::ControlsState::AlertSize::NONE);
+  setProperty("dm_fade_state", fmax(0.0, fmin(1.0, dm_fade_state+0.2*(0.5-(float)(dmActive)))));
```

Comment:
why use a property here? seems unnecessary

Reply:
how else do you set it? scene is const?